wrappers/services.go

```diff
@@ -199,7 +199,7 @@ func StartServices(apps []*snap.AppInfo, inter interacter, tm timings.Measurer)
 }
 
 // AddSnapServices adds service units for the applications from the snap which are services.
-func AddSnapServices(s *snap.Info, inter interacter) (err error) {
+func AddSnapServices(s *snap.Info, disabledSvcs []*snap.AppInfo, inter interacter) (err error) {
```

Comment:
I think it'd be a bit easier if we pass a list of app names here. Then it'd match the map retuned by `wrappers.ServicesEnableState` and what is kept in snap-setup/snap-state.
Eg:
```
func AddSnapServices(s *snap.Info, disabledSvcs []string, inter interacter) (err error) {
```

Reply:
I was trying to match the other methods here which all take AppInfo as an arg when referring to a service, but agreed it would be a fair bit simpler to just use`[]string` instead

Reply:
I was trying to match the other functions taking in `AppInfo` rather than a service name, but you're right that would simplify the logic. I will adjust this